src/v/features/feature_table.h

```diff
@@ -46,7 +46,7 @@ enum class feature : std::uint64_t {
     replication_factor_change = 0x2000,
     ephemeral_secrets = 0x4000,
     seeds_driven_bootstrap_capable = 0x8000,
-    tm_stm_cache = 0x16000,
+    tm_stm_cache = 0x10000,
```

Comment:
Can you update to using shift operator to remove extra cognitive load on picking the right value?

```
central_config = uint64_t(1) << 0,
consumer_offsets = uint64_t(1) << 1,
maintenance_mode = uint64_t(1) << 2,
mtls_authentication = uint64_t(1) << 3,
...
tm_stm_cache = uint64_t(1) << 16,
```